### PR TITLE
UD: Made slow-timer tick (time granularity) configurable.

### PR DESCRIPTION
**Pull request by alex--m**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10272/ for details (Mellanox internal link).

File: src/uct/ib/ud/base/ud_iface.c [hunk 1]
```diff
@@ -553,6 +560,8 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
 
     {"TIMEOUT", "5.0m", "Transport timeout",
      ucs_offsetof(uct_ud_iface_config_t, peer_timeout), UCS_CONFIG_TYPE_TIME},
+    {"SLOW_TIMER_TICK", "100ms", "Initial time granularity for resending trigger",
+     ucs_offsetof(uct_ud_iface_config_t, slow_timer_tick), UCS_CONFIG_TYPE_DOUBLE},
```
Comment (brminich) [inline src/uct/ib/ud/base/ud_iface.c]: need to be UCS_CONFIG_TYPE_TIME

File: src/uct/ib/ud/base/ud_iface.c [hunk 2]
```diff
@@ -437,6 +436,14 @@ UCS_CLASS_INIT_FUNC(uct_ud_iface_t, uct_ud_iface_ops_t *ops, uct_md_h md,
     self->config.check_grh_dgid  = config->dgid_check &&
                                    uct_ib_iface_is_roce(&self->super);
 
+    if (config->slow_timer_tick <= 0.) {
+        ucs_error("The slow timer tick should be > 0 (%lf)",
+                  config->slow_timer_tick);
+        return UCS_ERR_INVALID_PARAM;
+    } else {
+        self->config.slow_timer_tick = config->slow_timer_tick;
```
Comment (brminich) [inline src/uct/ib/ud/base/ud_iface.c]: could init  `iface->async.slow_tick` here, then no need to keep `self->config.slow_timer_tick`

**Comment by brminich** [comment]
ok to test

**Comment by shamisp** [comment]
Copyright is missing.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10329/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
@yosefe @alex--m The PR was replaced by #3703. I think we can close this PR, wdyt?

**Comment by alex--m** [comment]
@dmitrygladkov correct.

**Pull request closed without merging**